### DISCUSSION TITLE
Having Luck with Blazor FluentUI 🤩

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [microsoft/fluentui-blazor#1960] [Show and tell]
I am showing off my luck with your components on Twitter/X:

https://x.com/Mike_E_angelo/status/1784138798030332226

Saving a lot of time here with the controls.  As discussed in the above I estimated 2.5 days but got it done in 1. 👍🙏

**Comment by dvoituron** [comment]
Many thanks for your feedback. Awesome.
